Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleSceneView.swift

```diff
@@ -0,0 +1,262 @@
+                geoTrackingIsAvailable = try await checkGeoTrackingAvailability()
+            } catch {
+                self.error = error
+            }
+        }
+        .overlay(alignment: calibrationViewAlignment) {
+            if !calibrationViewIsHidden {
+                if !isCalibrating {
+                    Button {
+                        withAnimation {
+                            isCalibrating = true
+                        }
+                    } label: {
+                        Text("Calibrate")
+                            .padding()
+                    }
+                    .background(.regularMaterial)
+                    .clipShape(RoundedRectangle(cornerRadius: 10))
+                    .disabled(!initialCameraIsSet)
+                    .padding()
+                    .padding(.vertical)
+                }
+            }
+        }
+        .overlay(alignment: .bottom) {
+            if isCalibrating {
+                CalibrationView(viewModel: calibrationViewModel, isPresented: $isCalibrating)
+                    .padding(.bottom)
+            }
+        }
+        .overlay(alignment: .top) {
+            accuracyView
+        }
+    }
+    
+    /// A view that displays the horizontal and vertical accuracy of the current location datasource location.
+    @ViewBuilder
+    var accuracyView: some View {
+        if let currentLocation {
+            VStack {
+                Text("H. Accuracy: \(currentLocation.horizontalAccuracy.formatted(.number.precision(.fractionLength(2))))")
+                Text("V. Accuracy: \(currentLocation.verticalAccuracy.formatted(.number.precision(.fractionLength(2))))")
+            }
+            .multilineTextAlignment(.center)
+            .padding(8)
+            .frame(maxWidth: .infinity, alignment: .center)
+            .background(.regularMaterial)
+            .clipShape(RoundedRectangle(cornerRadius: 10))
+            .padding([.horizontal, .top])
+        }
+    }
+    
+    /// Sets the visibility of the calibration view for the AR experience.
+    /// - Parameter hidden: A Boolean value that indicates whether to hide the
+    ///  calibration view.
+    public func calibrationViewHidden(_ hidden: Bool) -> Self {
+        var view = self
+        view.calibrationViewIsHidden = hidden
+        return view
+    }
+    
+    /// Sets the alignment of the calibration view.
+    /// - Parameter alignment: The alignment for the calibration view.
+    public func calibrationViewAlignment(_ alignment: Alignment) -> Self {
+        var view = self
+        view.calibrationViewAlignment = alignment
+        return view
+    }
+    
+    /// Checks if GPS is providing the most accurate location and heading.
+    /// - Parameter locationManager: The location manager to determine the accuracy authorization.
+    /// - Returns: A Boolean value that indicates whether tracking is accurate.
+    private func checkTrackingCapabilities(_ locationManager: CLLocationManager) -> Bool {
```

Comment:
We may not need this method. I added it for discussion - we probably want a way to ensure the hardware is providing accurate updates before the AR experience starts.